### DISCUSSION TITLE
Autotune/Nightscout/Google Cloud

### DISCUSSION THREAD

**Discussion by dunvers** [community/community#56254] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi, I have Nightscout set up on Google Cloud and I'm trying to run AUTOTUNEWEB but I keep getting the error "The remote server returned an error: (401) Unauthorized". I have read many different documents and questions about Autotune and the only one that I thought might be the issue was changing AUTH_DEFAULT_ROLES to "readable" but I've done that and tried many times with renewed hope but I'm still getting the same error. If anybody has any ideas where the issue may lie or has any ideas what I could attempt to try and resolve the problem I would be very appreciative.
![Screenshot (43)](https://github.com/community/community/assets/53002076/f980bc47-8692-414d-9ef7-4b9822125740)

![Screenshot (42)](https://github.com/community/community/assets/53002076/437402fc-9584-4ff1-8d92-911439cbaaf7)

**Comment by dunvers** [comment, marked as answer]
RESOLVED

Although completely exiting the VM, and then going back into it to check that the variable had been changed it would appear that this is not enough. Although the variable was showing as changed the change hadn't taken effect for any API's associated with it. Restarting the VM ensured that the changes were updated in any API's and Autotune is now working. I hope this information helps others who may be having similar issues.

**Reply by Olololivia** [reply]
Hey, I ran into the same problem but I'm not as experienced with programming and VM stuff as you seem hahah. Would you be able to give an explanation of how and where you even change "AUTH_DEFAULT_ROLES to "readable" ".
I'd appreciate it a lot :,))
